filebeat/input/filestream/filestream.go

```diff
@@ -268,5 +280,6 @@ func (f *logFile) Close() error {
 	f.readerCtx.Cancel()
 	err := f.file.Close()
 	_ = f.tg.Stop() // Wait until all resources are released for sure.
+	f.log.Infof("Closed reader. Path='%s'", f.file.Name())
```

Comment:
How noisy does this log message have potential to be if there are 1000s of files? I recall we have regularly ended up with excessively noisy logging in that case.

Reply:
Thant should have been level debug, thanks.